File: sdkwork-backend-sdk-go/types/invitation_code_vo.go
package types

// 邀请码VO对象
type InvitationCodeVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Code string `json:"code"`
	CreatorUserId int `json:"creatorUserId"`
	Status string `json:"status"`
	ExpireTime string `json:"expireTime"`
	UsageLimit int `json:"usageLimit"`
	UsedCount int `json:"usedCount"`
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/invitation_code_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class InvitationCodeVO:
    """邀请码VO对象"""
    created_at: str = None
    updated_at: str = None
    code: str = None
    creator_user_id: int = None
    status: str = None
    expire_time: str = None
    usage_limit: int = None
    used_count: int = None
